### PR TITLE
Update lexicon...

### PR DESCRIPTION
**Pull request by thiollie**

### CODE REVIEW

**Review by bam241 — COMMENTED** [review]
"ordered" variables ?

File: LEXICON.md [hunk 1]
```diff
@@ -49,9 +49,23 @@ Represents the computationnal tool that processes the input and performing outpu
 
 ### Model simplifications
 
-	decay or not
-	depletion or not
+Model simplifications are used to for optimizing CPU calculation time without impacting the output. Simplifications have to be done according to targetted output variables.
```
Comment (bam241) [inline LEXICON.md]: with a reduce impact on the output ?

**Comment by bam241** [comment]
l46-48: I don't understand the mention of CYCLUS/CLASS/COSI in the Model, as far as I understand model represent depletion model or fabrication model, so way to predict cross section and solve Bateman equation, or a set of hypothesis and predictor to determine the fuel composition

**Comment by thiollie** [comment]
quoi quoi??? Tu veux t’bat’….??

**Comment by bam241** [comment]
anytime !

**Pull request merged**